src/index.html

```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__center"></div>
+      <div class="stopwatch__seconds stopwatch__seconds--5"></div>
+      <div class="stopwatch__seconds stopwatch__seconds--60"></div>
```

Comment:
for what need `--5` and `--60`? let's remove these numbers
```suggestion
      <div class="stopwatch__seconds stopwatch__seconds"></div>
      <div class="stopwatch__seconds stopwatch__minutes"></div>
```